curriculum/challenges/english/25-front-end-development/lecture-working-with-the-dom-click-events-and-web-apis/67336894ae148431a870694d.md

```diff
@@ -10,6 +10,105 @@ dashedName: what-is-the-dom-and-how-do-you-access-elements
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+What is the DOM, and how do you access elements?
+
+Let's learn about the DOM and why it's so important for web development.
+
+DOM stands for Document Object Model. It's a programming interface that lets us interact with HTML documents.
+
+With the DOM, you can add, modify, or delete elements on a webpage. You can even make your website interactive by making elements listen to and respond to events.
+
+In the DOM, an HTML document is represented as a tree of nodes. Each node represents an HTML element from the HTML document.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>DOM Example</title>
+  </head>
+  <body>
+    <h1>What is the DOM?</h1>
+    <h2>Let's learn about the DOM</h2>
+  </body>
+</html>
+```
+
+This is a diagram representing the basic DOM structure of our example. 
+
+```md
+               ┌──────────────────────────────────────────┐
+               │                Document                  │
+               └──────────────────────────────────────────┘
+                           │
+                           ▼
+               ┌──────────────────────────────────────────┐
+               │                   HTML                   │
+               └──────────────────────────────────────────┘
+                           │
+             ┌─────────────┼─────────────┐
+             ▼                           ▼
+    ┌────────────────┐            ┌────────────────┐
+    │     HEAD       │            │     BODY       │
+    └────────────────┘            └────────────────┘
+             │                           │
+    ┌────────┼────────┐         ┌────────┼────────┐
+    ▼                 ▼         ▼                 ▼
+┌─────────┐       ┌─────────┐ ┌─────────┐       ┌─────────┐
+│  TITLE  │       │  META   │ │   H1    │       │   H2    │
+└─────────┘       └─────────┘ └─────────┘       └─────────┘
```

Comment:
This is going to be a screen reader nightmare. I wonder if this would be better represented as an image with appropriate alt text?

Reply:
I can add a sentence explaining the diagram since we decided not to use images for transcripts

Reply:
I simplified the diagram so it should be more accessible now